### PR TITLE
Update ARM and ACTARM.

### LINKED ISSUE #315: Refactor ADAB
### What happened?

A bug happened!

### sessionInfo()

_No response_

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines.

### Security Policy

- [X] I agree to follow this project's Security Policy.

### PR DESCRIPTION
**Pull request by qiliu1013**
# Pull Request

Fixes #315

### CODE REVIEW

**Comment by qiliu1013** [comment]
I have read the CLA Document and I hereby sign the CLA

File: R/radab.R [hunk 1]
```diff
@@ -76,8 +76,8 @@ radab <- function(adsl,
   }
 
   # validate and initialize related variables
-  param_init_list <- relvar_init(param, paramcd)
-  unit_init_list <- relvar_init(param, avalu)
+  param_init_list <- random.cdisc.data:::relvar_init(param, paramcd)
+  unit_init_list <- random.cdisc.data:::relvar_init(param, avalu)
```
Comment (shajoezhu) [inline R/radab.R:80]:
```suggestion
  param_init_list <- relvar_init(param, paramcd)
  unit_init_list <- relvar_init(param, avalu)
```

**Comment by shajoezhu** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by shajoezhu — APPROVED** [review]
lgtm! thanks @qiliu1013

**Pull request merged**